sympy/series/tests/test_limits.py

```diff
@@ -737,6 +737,10 @@ def test_issue_18378():
 def test_issue_18442():
     assert limit(tan(x)**(2**(sqrt(pi))), x, oo, dir='-') == Limit(tan(x)**(2**(sqrt(pi))), x, oo, dir='-')
 
+def test_issue_18452():
+    assert limit(abs(log(x))**x, x, 0) == 1
+    assert limit(abs(log(x))**x, x, 0, "+") == 1
```

Comment:
By default the `dir` will be `+`, so the testcase having `dir` explicitly specified as `+` can be removed.

Reply:
> By default the `dir` will be `+` in this case, so the testcase having `dir` explicitly specified as `+` can be removed.